File: mvn-advent-of-code/src/test/java/advent/twenty_fifteen/Day16AuntSueTest.java
package advent.twenty_fifteen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.*;

public class Day16AuntSueTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day16AuntSue("/twenty_fifteen/Day16-AuntSue-input.txt");

        assertEquals(373, dp.getPart1Answer().intValue());
        assertEquals(260, dp.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_fifteen/Day16AuntSueTest.scala
package advent.twenty_fifteen

import org.scalatest.{FlatSpec, Matchers}

class Day16AuntSueTest  extends FlatSpec with Matchers {
  "2015 Day 16 - Input" should "calculate answers" in {
    val dp = new Day16AuntSue("/twenty_fifteen/Day16-AuntSue-input.txt")

    dp.part1Answer should be(373)
    dp.part2Answer should be(260)
  }
}
